model/user.go

```diff
@@ -23,5 +23,6 @@ import (
 )
 
 type User struct {
-	ID uuid.UUID
+	ID               uuid.UUID
+	NumberOfProjects int
```

Comment:
I feel this denormalization is not needed here, but I will let it go. I think you could infer number of projects from relation user -> projects.